### ISSUE TITLE
running darknet_video.py gives me: (563) cv::VideoWriter::open VIDEOIO(CV_IMAGES): raised OpenCV exception:

### ISSUE THREAD

**Issue by xrareselX** [AlexeyAB/darknet#7681]
Windows 10.
Python 3.8
opencv-python 4.5
So everything works fine when I test darknet with an image or a video from the cmd. However, when I try to run darknet_video.py in Pycharm, I get the following errors:
` [ERROR:0] global C:\Users\appveyor\AppData\Local\Temp\1\pip-req-build-kh7iq4w7\opencv\modules\videoio\src\cap.cpp (563) cv::VideoWriter::open VIDEOIO(CV_IMAGES): raised OpenCV exception: `

` OpenCV(4.5.1) C:\Users\appveyor\AppData\Local\Temp\1\pip-req-build-kh7iq4w7\opencv\modules\videoio\src\cap_images.cpp:207: error: (-215:Assertion failed) !filename.empty() in function 'cv::icvExtractPattern' `
Do you have any ideas what the problem might be?

**Comment by Ericasam2** [comment]
yes, I met the same problem, and i haven't yet worked out  a solution to handle the problem.
